src/js/stage/play.js

```diff
@@ -16,9 +16,14 @@ class PlayScreen extends Stage {
         var panel = new SideMenuContainer(game.viewport.width * 5/6, 0, game.viewport.width / 6, game.viewport.height);
         var gameHud = new GameHUD(0, 0, game.viewport.width * 5 / 6, game.viewport.height);
 
-        var game_map = new me.Sprite(450,350, 
+        const width = game.viewport.width;
```

Comment:
Move width and height to top of file AND replace all instances of game.viewport.width and game.viewport.height with the variables (e.g. game.viewport.width is replaced with width)